modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1189,6 +1189,50 @@ class CV_EXPORTS_W TrackerTLD : public Tracker
   BOILERPLATE_CODE("TLD",TrackerTLD);
 };
 
+/** @brief KCF is a novel tracking framework that utilizes properties of circulant matrix to enhance the processing speed.
+ * This tracking method is an implementation of @cite KCF_ECCV which is extended to KFC with color-names features (@cite KCF_CN).
+ * The original paper of KCF is available at <http://home.isr.uc.pt/~henriques/circulant/index.html>
+ * as well as the matlab implementation. For more information about KCF with color-names features, please refer to
+ * <http://www.cvl.isy.liu.se/research/objrec/visualtracking/colvistrack/index.html>.
+ */
+class CV_EXPORTS_W TrackerKCF : public Tracker
+{
+ public:
+  enum MODE {GRAY, CN, CN2};
+  struct CV_EXPORTS Params
+  {
+    Params();
+
+    /**
+     * \brief Read parameters from file, currently unused
+     */
+    void read( const FileNode& /*fn*/ );
+
+    /**
+     * \brief Read parameters from file, currently unused
+     */
+    void write( FileStorage& /*fs*/ ) const;
+
+    double sigma;                 //!<  gaussian kernel bandwidth
```

Comment:
I personally prefer the documentation for attributes in the line before their declaration. One reason why: say that you rename the attribute with the longest name (`output_sigma_factor` in this case). As it is done now, in order to keep all the comments aligned, you would need to update also all the other lines with the rest of the attributes.

Anyway, for me it is also fine if you want to keep it this way.